packages/in_app_purchase/lib/src/billing_client_wrappers/purchase_wrapper.dart

```diff
@@ -89,6 +90,21 @@ class PurchaseWrapper {
   /// Note though that verifying a purchase locally is inherently insecure (see
   /// the article for more details).
   final String originalJson;
+
+  /// Generate a [PurchaseDetails] object based on the transaction.
+  ///
+  /// [PurchaseDetails] is Used to represent a purchase detail for unified iOS and Android API.
```

Comment:
nit: 

```suggestion
  /// [PurchaseDetails] is used to represent a purchase for the unified payment APIs.
```

This conversion has to be one way only too, which I think is worth mentioning.